eval/builtin_fn_io_test.go

```diff
@@ -19,17 +15,19 @@ func TestBuiltinFnIO(t *testing.T) {
 		That(`print "a\nb" | slurp`).Puts("a\nb"),
 		That(`print "a\nb" | from-lines`).Puts("a", "b"),
 		That(`print "a\nb\n" | from-lines`).Puts("a", "b"),
-		That(`echo '{"k": "v", "a": [1, 2]}' '"foo"' | from-json`).Puts(
```

Comment:
Did you remove this test case by mistake?

Reply:
No, the test case I added made it redundant.

Reply:
I don't see what you mean; you haven't added a test case against the output of `from-json`.

Reply:
Does the test case that tests that going from json, then back to the same json not cover it?

Reply:
No. That only tests that `from-json` and `to-json` are inverse operations (under certain conditions). If you both `from-json` and `to-json` are simply identity functions, the test you added would pass.